Scripts/Set-Output.ps1

```diff
@@ -0,0 +1,11 @@
+param (
+    [Parameter(Mandatory = $true, Position = 0)]
+    [string]
+    $Key,
+
+    [Parameter(Mandatory = $true, Position = 1)]
+    [string]
+    $Value
+)
+
+return Write-Output "$Key=$Value" >> $GITHUB_OUTPUT
```

Comment:
This is duplicated with Write-CacheConfiguration.ps1. Also, check all affected workflows: Is this script really available in each of them? Because it's not loaded.

Reply:
The only workflows utilizing this are the `create-jira-issues`, `publish-nuget` and `deploy-to-azure`, I'm not sure if I can test it with `publish-nuget` or `deploy-to-azure`, but I will create a test issue with `create-jira-issues`  to check.

Reply:
You can make dummy runs of these workflows in each case (like you did with `create-jira-issues`). The full run doesn't need to succeed, but it has to pass the first point where this new script is used.

BTW unless you load the _Scripts_ folder at the beginning of the action (not workflow) then calls to this script won't find it. See e.g.: https://github.com/Lombiq/GitHub-Actions/blob/dev/.github/actions/create-jira-issues-for-community-activities/action.yml#L25

Reply:
Managed to make this function work [here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/3365226333/jobs/5580478419) finally. The gaijira login wouldn't start if the set-output function wouldn't work.

Reply:
Great!

Reply:
I still don't see the Scripts folder being loaded for all affected actions.